alchemist/alchemist-loading/src/main/resources/it/unibo/alchemist/loader/YamlSyntax.properties

```diff
@@ -11,13 +11,16 @@ aggregators = aggregators
 concentration = concentration
 conditions = conditions
 contents = contents
+data-rate = data-rate
 default = default
 displacements = displacements
 environment = environment
+error-rate = error-rate
 export = export
 formula = formula
 in = in
 incarnation = incarnation
+is-udp = is-udp
```

Comment:
I'd prefer to have the protocol specified by text. It's ok if only UDP and TCP are supported right now.